### DISCUSSION TITLE
Issues launching the app

### DISCUSSION THREAD

**Discussion by kingbonj** [death7654/Chrultrabook-Tools#78] [Q&A]
First time I ran this on Deb 12 it seemed to work well, then for some reason  opening the appimage seems to start a process in top but I am no longer able to see the app window, it closes immediately after opening. I can see new directories in '~/.cache' , as well as  '/tmp' but there doesnt seem to be a way to interact with the app at all. Ive redownloaded to see what I can do, was hoping there would be some config file I can delete and see if that helps. Any ideas?

**Comment by death7654** [comment]
Please create an issue in the issues tab.
